### DISCUSSION TITLE
很好的项目，可否考虑输出仅包含warp部分的信息，即提供proxy-providers

### DISCUSSION THREAD

**Discussion by hahach** [vvbbnn00/WARP-Clash-API#38] [Ideas]
非常好的项目，看了下，普通小白用应该很好了。我提的这个需求是方便有自己规则的人的。
我看到项目示例https://subs.zeabur.app/ 里提供了仅代理的输出，但是里面还是包含了策略组（proxy-groups）的设定，而且非常多。

我想自己配置proxy-groups，因此只想获取warp代理这部分。这个需求目前可以通过https://proxy-provider-converter.vercel.app/ 这类服务实现。转换后的proxy-providers:里就有我需要的仅含有warp代理配置的信息了。
不过还是希望能直接通过本项目直接提供，而不是转手，减少不必要的麻烦。

**Comment by hahach** [comment]
看了下代码，我自己改了下 services/subscription.py 里面的输出格式，现在已经实现我的需求了。

就是有个奇怪的地方没搞懂，我明明给了 DO_GET_WARP_DATA=False 但是容器启动后还是会刷流量。我现在的做法是直接把run.sh里的 python3 app.py background& 注释掉了。不清楚是不是bug

**Comment by vvbbnn00** [comment]
> 非常好的项目，看了下，普通小白用应该很好了。我提的这个需求是方便有自己规则的人的。 我看到项目示例https://subs.zeabur.app/ 里提供了仅代理的输出，但是里面还是包含了策略组（proxy-groups）的设定，而且非常多。
> 
> 我想自己配置proxy-groups，因此只想获取warp代理这部分。这个需求目前可以通过https://proxy-provider-converter.vercel.app/ 这类服务实现。转换后的proxy-providers:里就有我需要的仅含有warp代理配置的信息了。 不过还是希望能直接通过本项目直接提供，而不是转手，减少不必要的麻烦。

了解您的需求，该功能将在后续更新

**Comment by vvbbnn00** [comment]
已经支持该功能：https://github.com/vvbbnn00/WARP-Clash-API/pull/49
